docs/GettingStarted.md

```diff
@@ -144,6 +145,57 @@ Where `options` is an optional `Hash` that accepts the following parameters:
 
 ## Other libraries
 
+### GraphQL
+
+*Version 1.7.9+ supported*
+
+The GraphQL integration activates instrumentation for GraphQL queries. To activate your integration, use the ``Datadog.configure`` method:
+
+    # Inside Rails initializer or equivalent
+    Datadog.configure do |c|
+      c.use :graphql,
+            service_name: 'graphql',
+            schemas: [YourSchema]
+    end
+
+    # Then run a GraphQL query
+    YourSchema.execute(query, variables: {}, context: {}, operation_name: nil)
+
+Where `options` is an optional `Hash` that accepts the following parameters:
+
+| Key | Description | Default |
+| --- | --- | --- |
+| ``service_name`` | Service name used for `graphql` instrumentation | ``ruby-graphql`` |
+| ``schemas`` | Optional. Array of `GraphQL::Schema` objects which to trace. Tracing will be added to all the schemas listed, using the options provided to this configuration. If you supply your schema in this option, do not add `use(GraphQL::Tracing::DataDogTracing)` to the schema's definition, to avoid double traces. If you do not supply this option, you will need to individually configure your own schemas (see below.) | ``[]`` |
+| ``tracer`` | A ``Datadog::Tracer`` instance used to instrument the application. Usually you don't need to set that. | ``Datadog.tracer`` |
+
+##### Manually configuring GraphQL schemas
+
+If you have multiple schemas, and/or you prefer to individually configure the tracer settings for each,
+in the schema definition, you can add the following:
+
+```
+YourSchema = GraphQL::Schema.define do
+  use(
+    GraphQL::Tracing::DataDogTracing,
+    service: 'graphql',
+    tracer: Datadog.tracer
```

Comment:
I would remove `tracer` from the example. Let's keep it minimal.

Reply:
This is pretty important if you're manually instrumenting. I'd recommend keeping it especially since its not documented anywhere else in either `ddtrace` or `graphql`. But if you're convinced it doesn't add value, I'll remove it.

Reply:
`Datadog.tracer ` is always the de-facto `tracer` you have to use. Initializing a new tracer and set it as tracer for GraphQL (while using other integrations) will only create instrumentation issues.

Reply:
I'll remove this.